src/components/30-organisms/commercial-hero-banner/story.js

```diff
@@ -55,6 +58,40 @@ export const CommercialHeroBanner = () => {
         >GET A QUOTE</axa-button-link
       >
     </axa-commercial-hero-banner>
+    <br />
+    <br />
+    <axa-commercial-hero-banner
+      variant="${variant}"
+      imagesource="${imageSource}"
+    >
+      <header slot="title">
+        <p>This example shows specific picture classes</p>
+        <h1>Drive with peace of mind</h1>
+      </header>
+      <div slot="content" class="checkmarks">
+        <div class="checkmark">
+          <axa-icon class="checkmark-icon" icon="check-circle" size="auto"></axa-icon>
+          <span class="checkmark-text">Extra Cookie</span>
+        </div>
+        <div class="checkmark">
+          <axa-icon class="checkmark-icon" icon="check-circle" size="auto"></axa-icon>
+          <span class="checkmark-text">Awesome People</span>
+        </div>
+        <div class="checkmark">
+          <axa-icon class="checkmark-icon" icon="check-circle" size="auto"></axa-icon>
+          <span class="checkmark-text">Best in class Service</span>
+        </div>
+      </div>
+      
+      <axa-button-link
+        href="https://axa.ch"
+        slot="button"
+        variant="red"
+        size="large"
+        >GET A QUOTE</axa-button-link
+      >
+    </axa-commercial-hero-banner>      
+    </div>
```

Comment:
Having 2 elements within the same story is a bit weird. The main story should not be polluted with several variants. Better use your new one as main and adapt the tests.

Reply:
I used the new one and removed from test the parts related to disclaimer. All tests pass now.

Reply:
I also used prettier to format the code in commercial-hero-banner/story.js but then I realised that you already did that and pushed.